src/main/scala/nl/knaw/dans/easy/transform/XmlTransformation.scala

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) 2019 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.transform
+
+import java.net.URI
+
+import nl.knaw.dans.easy.transform.AccessRights.AccessRights
+
+import scala.xml.transform.{ RewriteRule, RuleTransformer }
+import scala.xml.{ Attribute, Elem, Node, Null }
+
+class XmlTransformation() {
```

Comment:
No constructor params, no mocking required, so lets make this an `object`:

```suggestion
object XmlTransformation {
```

Reply:
right. done